src/mesh/region.cc

```diff
@@ -146,6 +146,12 @@ Region RegionDB::add_region(unsigned int id, const std::string &label) {
         THROW(ExcNonuniqueLabel() << EI_Label(label) << EI_ID(id) << EI_IDOfOtherLabel(it_label->get_id()) );
     }
 
+    OnlyIDIter it_only_id = region_set_.get<OnlyID>().find(id);
```

Comment:
Why we need this if regions for region sets are added in close()?
However in order to simplify operations for creating region sets (union, etc.) we should
create region sets for regions just after reading the mesh.

Possibly we can just rename this method to rename_region, corresponding to the operation to name the region.